indico/modules/events/management/templates/action_menu.html

```diff
@@ -36,4 +68,15 @@
             return true;
         }
     });
+
+    var $clonesTarget = $('#action-menu-clones-target');
```

Comment:
wrap this whole script block in an IIFE with strict mode. right now you create globals. also, you forgot a `console.log` there ;)